### PR TITLE
Add Self-Description for Dataspace Authority

### PR DESCRIPTION
**Pull request by bscholtes1A**
## What this PR changes/adds

Add Self-Description for Dataspace Authority

## Why it does that

Dataspace authority should have a Self-Description.

## Linked Issue(s)

#19 

## Checklist

- [ ] added appropriate tests?
- [ ] performed checkstyle check locally?
- [ ] added/updated copyright headers?
- [ ] documented public classes/methods?
- [ ] added/updated relevant documentation?
- [ ] added relevant details to the changelog? (_skip with label `no-changelog`_)
- [ ] formatted title correctly?

### CODE REVIEW

**Review by algattik — APPROVED** [review]

**Review by marcgs — APPROVED** [review]

File: system-tests/docker-compose.yml [hunk 1]
```diff
@@ -157,12 +157,14 @@ services:
...
       EDC_CONNECTOR_NAME: registration-service
       EDC_IDENTITY_DID_URL: did:web:did-server:registration-service
+      EDC_SELF_DESCRIPTION_DOCUMENT_PATH: /resources/self-description/registration-service/sdd.json
       EDC_VAULT: /resources/vault/registration-service/registration-service-vault.properties
       EDC_KEYSTORE: /resources/vault/registration-service/registration-service-keystore.jks
       EDC_KEYSTORE_PASSWORD: test123
       EDC_ERROR_RESPONSE_VERBOSE: "true"
     ports:
       - "8184:8184"
+      - "8185:8181"
```
Comment (marcgs) [inline system-tests/docker-compose.yml:167]: Just for understanding...  why was this required?

Comment (bscholtes1A) [reply system-tests/docker-compose.yml:167]: The identity hub endpoint of the registration service is used in the `IdentityHubIntegrationTest`
